cmd/buildctl/build.go

```diff
@@ -228,7 +228,7 @@ func build(clicontext *cli.Context) error {
 			}
 		}
 		// not using shared context to not disrupt display but let is finish reporting errors
-		return progressui.DisplaySolveStatus(context.TODO(), c, os.Stdout, displayCh)
+		return progressui.DisplaySolveStatus(context.TODO(), "", c, os.Stdout, displayCh)
```

Comment:
Rather than require `""` in callers who don't care I was debating making the last  argument a `opts ...WithOpt` style thing. I went with the simple/easy way for the first cut though.